pkg/scheduler/scheduler.go

```diff
@@ -845,11 +885,7 @@ func (ks *KubernetesScheduler) recoverTasks() error {
 		defer ks.Unlock()
```

Comment:
Second round is fine. I thought I saw that you had already started, so I
was just pointing out the rest.

On Mon, Apr 13, 2015 at 7:11 AM, Joerg Schad <notifications@github.com>
wrote:

> In pkg/scheduler/scheduler.go
> <https://github.com/mesosphere/kubernetes-mesos/pull/220#discussion_r28229983>
> :
>
> > @@ -845,11 +885,7 @@ func (ks *KubernetesScheduler) recoverTasks() error {
> >  		defer ks.Unlock()
>
> I actually planned to remove the redundant locks in a second round. Is
> that ok or do you prefer this in the same patch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/kubernetes-mesos/pull/220/files#r28229983>.
>



-- 
James DeFelice
585.241.9488 (voice)
650.649.6071 (fax)